javascript/javascript2/week1/script1.js

```diff
@@ -0,0 +1,26 @@
+const spiritAnimalNames = [ "Fish", "Fig","Ant", "Bear", "Frog", "Rabbit", "Giraffe", "Elephant", "Bat", "Polar"];
+                          
+var heading = document.createElement("h1");
```

Comment:
f I think you could just add this two elements in your html file, but it is a good exercise to learn how to create elements in js too :) 

Instead of using `var` keyword to create variables in this case you should use `const` to do that as we will never reasing this variable again. it holds a reference to our element. 
The interesitng thing is that event if we declare a variable as a `const` and our variable is of reference type (object or array) we can still add or asign properties to it or push elements to the array sotred as a const. 
Here's an example
```
const object = {key: 'value1'}
obj.key = 'value2' //that is ok we can do that - we can edit an object stored in a `const`
object = {otherKey: 'otherValue'} //that will cause an error - we are trying to reaasing a const
```